CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+#How to contribute
+Alfresco's maturity comes from the contributions of its community. Contributing back to Alfresco can be a bit confusing as Alfresco is a collection of separate open source projects. Each open source project officially sponsored by Alfresco has a page in this space detailing its governance policy and the method for reporting issues and submitting contributions.
```

Comment:
I'm not sure what is meant by "page in this space", and "officially sponsored by Alfresco" sounds weird too - are there some Alfresco projects that we don't officially sponsor?

Reply:
Good point, though i'd be tempted to leave this in as a first draft and we can get together with ether Richard or Derek to discuss the wording (this was taken from here: https://community.alfresco.com/docs/DOC-6269-submitting-contributions). If we are going to change this we may want to change it there also.

Reply:
I think it's easier to take the paragraph out for now. If people are reading the records-management contribution guidelines then I think it's obvious which project they're interested in contributing to.  You could just change the next sentence to be something like: "For an overview about the records-management project see here."